distribution/api-docs-dist/pom.xml

```diff
@@ -125,7 +125,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-deploy-plugin</artifactId>
                 <configuration>
-                    <skip>true</skip>
+                    <skip>snapshots</skip>
```

Comment:
What's the reason behind this change? We publish api-docs for the nightly releases, which are snapshots, and this would stop that